examples/inline_types/src/types.ts

```diff
@@ -10,17 +10,18 @@ import {
     nat,
     nat64,
     query,
+    Record,
     update,
     Variant
 } from 'azle';
 
-export type User1 = {
+export type User1 = Record<{
     id: string;
     job: {
```

Comment:
We should discuss the inline records having the `Record` wrapping as well, just like inline variants.

Reply:
I've updated the parser to handle/require this.